### DISCUSSION TITLE
Question marks in the menu?

### DISCUSSION THREAD

**Discussion by gerroon** [LazyVim/LazyVim#2098] [Q&A] [answered]
Hi

Why do I have these question marks there? Am I missing a font or icon?

![image](https://github.com/LazyVim/LazyVim/assets/8519469/a17148f8-12b9-4ea1-bbe8-43fd1801368e)

**Comment by dimas-cyriaco** [comment, marked as answer]
You need a Nerd Font: https://www.lazyvim.org/#%EF%B8%8F-requirements

**Reply by gerroon** [reply]
Thanks, which one? There are alot https://www.nerdfonts.com/font-downloads?

**Reply by neuromaancer** [reply]
Anyone you like!

**Reply by dpetka2001** [reply]
If you use `kitty` as your terminal emulator, you can use the option `symbol_map` to define the font for the symbols ([NerdfontsSymbolsOnly](https://github.com/ryanoasis/nerd-fonts/releases/download/v3.1.1/NerdFontsSymbolsOnly.zip)) and you can use any font you like for the rest. Take a look at folke's [dotfiles](https://github.com/folke/dot/blob/1007fc65738ad1f7a3e9c91432430017a6878378/config/kitty/kitty.conf#L13) for kitty (that's where I took the idea for this approach for myself as well). Maybe other terminal emulators provide this functionality as well, so you'd have to read the docs of the terminal emulator you use to achieve the same.

**Reply by gerroon** [reply]
Thanks guys, that was super helpful.

**Reply by TheGr8Blue** [reply]
what do i do once i download? will it happen automatically or do i select the font somehow so that lazyvim uses those icons
